ReactAndroid/src/main/java/com/facebook/react/common/ViewHelperMethods.java

```diff
@@ -0,0 +1,22 @@
+package com.facebook.react.common;
+
+import android.view.View;
+
+/**
+ * Various helper methods for {@link View}
+ */
+public class ViewHelperMethods {
```

Comment:
`ViewHelpers` or `ViewUtils`. Can you check which naming is more common in the codebase? Also, is there an existing class where this could live?